### DISCUSSION TITLE
Upcoming Istio v1.22.7, v1.23.4, and v1.24.2 Security Releases

### DISCUSSION THREAD

**Discussion by blancobrenden** [istio/istio#54282] [Announcements]
Hello Istio Community,

The Istio maintainers would like to announce the forthcoming security release of Istio v1.22.7, v1.23.4, and v1.24.2.

The release will be available on December 18th 2024, and addresses some security vulnerabilities in Envoy, in which the highest CVE score is 7.5. No further details or patches will be made available in advance of the release.

Thanks,
Brenden Blanco (on behalf of the Istio Product Security Work Group)

**Comment by justinpettit** [comment]
LTGM
